great-idea/css/about.css

```diff
@@ -0,0 +1,161 @@
+/* http://meyerweb.com/eric/tools/css/reset/
+   v2.0 | 20110126
+   License: none (public domain)
+*/
+
```

Comment:
For some reason your styles aren't applying to your about page

Reply:
hmm, I got them working  on my end